test/catalog/catalog_test.cpp

```diff
@@ -251,6 +251,20 @@ TEST_F(CatalogTests, DroppingDatabase) {
   txn_manager.CommitTransaction(txn);
 }
 
+TEST_F(CatalogTests, CheckDatabaseExists) {
+  auto exists = catalog::Catalog::GetInstance()->CheckDatabaseExists("EMP_DB");
+  EXPECT_EQ(false, exists);
+
+  auto &txn_manager = concurrency::TransactionManagerFactory::GetInstance();
+  auto txn = txn_manager.BeginTransaction();
+  catalog::Catalog::GetInstance()->CreateDatabase("EMP_DB", txn);
+
+  exists = catalog::Catalog::GetInstance()->CheckDatabaseExists("EMP_DB");
+  EXPECT_EQ(true, exists);
```

Comment:
This would be false as the transaction created at line 259 has not been committed yet and therefore, the new database would not be visible to others.

Reply:
Snap! This is one of the reasons the tests are failing. Thanks for pointing out.